src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1314,6 +1314,15 @@
   <data name="PreferTypedStringBuilderAppendOverloadsRemoveToString" xml:space="preserve">
     <value>Remove the ToString call</value>
   </data>
+  <data name="PreferStringContainsOverIndexOfDescription" xml:space="preserve">
+    <value>Calls to 'String.IndexOf' where the result is compared to 0 or -1 can be replaced by 'String.Contains'</value>
+  </data>
+  <data name="PreferStringContainsOverIndexOfMessage" xml:space="preserve">
+    <value>Use 'String.Contains' instead of 'String.IndexOf' to improve readability</value>
```

Comment:
Do we prefer using `String` instead of `string` in user facing message string? Just a question, will let you and the PM team decide.

Reply:
I'll go ahead and merge what I have now and change it in the fixer if we need to